.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: java
+
+dist: trusty
+
+jdk:
+- oraclejdk8
+
+addons:
+    sonarcloud:
+        organization: "jfoenixadmin"
+        token:
+            secure: "**************************"
```

Comment:
This token must be changed with yours.
you can generate through https://sonarcloud.io/account/security/